### ISSUE TITLE
Squiz.Functions.FunctionDeclarationArgumentSpacing reports line break as 0 spaces between parenthesis

### ISSUE THREAD

**Issue by ComiR** [squizlabs/PHP_CodeSniffer#3273]
**Describe the bug**
`Squiz.Functions.FunctionDeclarationArgumentSpacing.SpacingBetween` has a false positive when there is a line break between empty function declaration parentheses.

**Code sample**
```php
<?php
function someVeryLooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooFunctionName(
): WithAnEvenLoooooooooooooooooooooooooooooooooooooooooooooooooongerClassNameAsReturnType {
    // Very long function name with a very long class name as return type as an example
    // why one would introduce a line break here.
}
```

**To reproduce**
1. Create a file called `test.php` with the code sample above.
2. Run `phpcs --standard=Squiz --sniffs=Squiz.Functions.FunctionDeclarationArgumentSpacing test.php`
3. See error message displayed
```
Expected 0 spaces between parenthesis of function declaration; 0 found
```

**Expected behavior**
No error.

**Versions (please complete the following information):**
 - OS: MacOS 10.15
 - PHP: 7.4
 - PHPCS: 3.5.8
 - Standard: Squiz

**Additional context**
If this is intended, there should be an option to allow line breaks.
If I put a comment between the parentheses, `Squiz.Functions.MultiLineFunctionDeclaration.FirstParamSpacing` would be the error from that. Maybe that should be fixed too.

**Comment by jrfnl** [comment]
The sniff works as intended. May not be the right sniff for you or alternatively, run the sniff with `-s` to see the exact error code and you could choose to exclude that error code from your custom ruleset or use it in an inline ignore annotation.

**Comment by ComiR** [comment]
So this sniff isn't only supposed to detect sth. like `function( )` but allow `function(\n)`?

**Comment by gsherwood** [comment]
The bug here was that the error message was wrong, and so very confusing. The error message now reports that a newline was found and not 0 spaces.

The sniff is working as expected, but feel free to put in a feature request if you think an option to ignore newlines would be good, or do what @jrfnl suggested and exclude the error message in a custom ruleset.

I'm closing this report as the bug in the message is fixed, but feel free to report more and thanks for reporting this.
